src/legacy/ts-jest-transformer.spec.ts

```
@@ -432,6 +432,21 @@ describe('TsJestTransformer', () => {
       expect(omitLeadingWhitespace(result.code)).toContain(expectedResult)
     })
 
+    it('should transpile import-only ESM node_modules file', () => {
+      const result = tr.process(
+        `import './side-effect.mjs';`,
+        'my-project/node_modules/foo.mjs',
+        baseTransformOptions,
+      )
```

ESLint

[error] Replace `⏎········`import·'./side-effect.mjs';`,⏎········'my-project/node_modules/foo.mjs',⏎········baseTransformOptions,⏎······` with ``import·'./side-effect.mjs';`,·'my-project/node_modules/foo.mjs',·baseTransformOptions`

(prettier/prettier)